services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -87,7 +91,10 @@ class WriteMetadataActor(override val dbBatchSize: Int, override val dbFlushRate
 }
 
 object WriteMetadataActor {
-  def props(dbBatchSize: Int, flushRate: FiniteDuration): Props = Props(new WriteMetadataActor(dbBatchSize, flushRate)).withDispatcher(ServiceDispatcher)
+  val MetadataWritePath = MetadataServiceActor.MetadataInstrumentationPrefix.::("writes")
```

Comment:
is there a more Englishy version of that method that might be more readable?

Reply:
Two suggestions:

`.doubleColon`?
`.colonColon`?

Reply:
Not that I see, I could do `MetadataServiceActor.MetadataInstrumentationPrefix.++(NonEmptyList.one("writes"))` but that seems unnecessary.